README.md

```diff
@@ -30,13 +30,19 @@ To run the spec generator locally and preview the formatted spec website, use th
 
 ```
```

Comment:
You have to install mdBook as well right? We should mention that and the specific version we expect you to use and link to the how to install it.

Reply:
We should also have the prereq of https://github.com/Michael-F-Bryan/mdbook-linkcheck

I wonder if there is a way to make it such that just a `cargo install` can install both mdbook and mdbook-linkcheck. Then we just have the rust requirement and `cargo install` and `npm install`.

Reply:
Yes this is a good place to setup linting for rust projects we can use the same outcome in mrc repo

Reply:
I'll add the commands to readme for now. I believe we can set up cargo config to list the packages